src/core/annotation.js

```diff
@@ -941,6 +941,54 @@ class Annotation {
     return null;
   }
 
+  get hasSomeText() {
+    return false;
+  }
+
+  async extractTextContent(evaluator, task) {
+    if (!this.hasSomeText || !this.appearance) {
+      return "";
+    }
+
+    const resources = await this.loadResources(
+      ["ExtGState", "Font", "Properties", "XObject"],
+      this.appearance
+    );
+
+    const text = [];
+    const buffer = [];
+    const sink = {
+      desizedSize: Math.Infinity,
+      ready: true,
+
+      enqueue(chunk, size) {
+        for (const item of chunk.items) {
+          buffer.push(item.str);
+          if (item.hasEOL) {
+            text.push(buffer.join(""));
+            buffer.length = 0;
+          }
+        }
+      },
+    };
+
+    await evaluator.getTextContent({
+      stream: this.appearance,
+      task,
+      resources,
+      includeMarkedContent: true,
+      combineTextItems: true,
+      sink,
+      viewBox: [-Infinity, -Infinity, Infinity, Infinity],
```

Comment:
Does it simply not matter what these values are, or do you perhaps want to pass in the `view`-property (on the `Page`-instance) similar to how the "normal" text-extraction does it?